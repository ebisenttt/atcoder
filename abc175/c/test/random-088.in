994152944098635 138813176949803 610045593099873
