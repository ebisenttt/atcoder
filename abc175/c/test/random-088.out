384107350998762
